debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -102,6 +109,7 @@ public void shouldReceiveChangesForInsertsWithDifferentDataTypes() throws Except
     }
 
     @Test
+    @ShouldFailWhen(DecoderDifferences.AreQuotedIdentifiersUnsupported.class)
```

Comment:
Nice pattern! Thinking of it, these subtle difference should be mentioned in the docs, too.

Reply:
Do you mean difference between decoders? It will be of course documented in Postgres connector documentation.

Reply:
Yes, exactly.